runtime.txt

```diff
@@ -0,0 +1 @@
+python-3.10.1
```

Comment:
didn't we configure our project as python 3.9?

Reply:
Yes we did
![image](https://user-images.githubusercontent.com/61323123/170820243-5e29b68a-e53a-4d50-854d-b8e067fb437f.png)
Here are the supported python versions by Heroku, Do you think we should use 3.9.13?
As far as I see it works well with 3.10

Reply:
if it works it ain't broken I suppose
but if it also works with the 3.9.13 I think it makes more sense

Reply:
👍